### PR TITLE
[#406] authmate: update default bearer rules

### LINKED ISSUE #406: Reconsider default EACL in authmate
https://github.com/nspcc-dev/neofs-s3-gw/blob/a0a04a73bdf2932730ff07b11c01de5ba4842ef7/authmate/authmate.go#L311

This rule does not deny anything. It is not what we expect from default rule. Instead it should have 8 records: 1 allow + 6 denies for each object service verb.

### PR DESCRIPTION
**Pull request by KirillovDenis**
closes #406 

Signed-off-by: Denis Kirillov <denis@nspcc.ru>

### CODE REVIEW

File: authmate/authmate.go [hunk 1]
```diff
@@ -320,16 +320,28 @@ func buildEACLTable(eaclTable []byte) (*eacl.Table, error) {
...
 
+	for _, rec := range restrictedRecords() {
+		table.AddRecord(rec)
+	}
+
 	return table, nil
 }
 
+func restrictedRecords() (records []*eacl.Record) {
+	for op := eacl.OperationGet; op <= eacl.OperationRangeHash; op++ {
```
[the remark this answers is not in the document]
Comment (KirillovDenis) [reply authmate/authmate.go:334]: I decided to use full restricted rules so we don't have to worry about what we put at the beginning of the eacl table.

[the remark this answers is not in the document]
Comment (masterSplinter01) [reply authmate/authmate.go:334]: In that case shouldn't we delete lines 320-323? They look confusing.

[the remark this answers is not in the document]
Comment (masterSplinter01) [reply authmate/authmate.go:334]: And could you update docs about bearer-rules in authmate?

[the remark this answers is not in the document]
Comment (KirillovDenis) [reply authmate/authmate.go:334]: > In that case shouldn't we delete lines 320-323? They look confusing.

I tried to explain what I mean in docs.

**Review by masterSplinter01 — APPROVED** [review]

**Pull request merged**